qadence/models/qnn.py

```diff
@@ -43,6 +44,7 @@ def __init__(
         self,
         circuit: QuantumCircuit,
         observable: list[AbstractBlock] | AbstractBlock,
+        inputs: tuple[sympy.Basic | str, ...] | None = None,
         transform: Callable[[Tensor], Tensor] = None,  # transform output of the QNN
         backend: BackendName = BackendName.PYQTORCH,
```

Comment:
can the `inputs` rely on `Parameter` (`FeatureParamater`, for instance)? I'm thinking in terms of custom feature map circuits

Reply:
`Parameter` is subclassing `sympy.Basic`, so yes:)